internal/client/login.go

```diff
@@ -48,14 +48,19 @@ func (client Client) GetServiceTokenDetailsV2() (GetServiceTokenDetailsResponse,
 }
 
 func (client Client) OidcMachineIdentityAuth() (string, error) {
-	authJwt := os.Getenv(INFISICAL_AUTH_JWT_NAME)
+	tokenEnvironmentName := client.Config.OidcTokenEnvName
+	if tokenEnvironmentName == "" {
+		tokenEnvironmentName = "TFC_WORKLOAD_IDENTITY_TOKEN" // default
+	}
+
+	authJwt := os.Getenv(tokenEnvironmentName)
```

Comment:
I think for this not to be breaking for existing pipelines, we should try and read the INFISICAL_AUTH_JWT_NAME env first and only resort to the `OidcTokenEnvName` if no value is found there